src/bandwidthRtc.ts

```diff
@@ -33,14 +32,17 @@ class BandwidthRtc {
    */
   async connect(authParams: RtcAuthParams, options?: RtcOptions) {
     const jwtPayload = jwt_decode<JwtPayload>(authParams.deviceToken);
-    if (jwtPayload.v?.toLowerCase() === "v3") {
-      logger.info("Using device API version 3");
-      this.delegate = new BandwidthRtcV3(this.logLevel);
-    } else {
-      logger.info("Using device API version 2");
-      this.delegate = new BandwidthRtcV2();
+    const rtcVersion = jwtPayload.v?.toLowerCase();
+    switch (rtcVersion) {
+      case "v3":
+      default: {
+        logger.info("Using device API version 3");
+        this.delegate = new BandwidthRtcV3(this.logLevel);
+      }
+    }
+    if (!this.delegate) {
```

Comment:
By defaulting to the V3 delegate (in the switch block) I don't think we'll ever hit this case. It might be better to throw this error in the default block in the swtich